person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+  private :of_age?
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
+  public :can_use_services?
```

Comment:
- [ ] The can_use_services functionality will not work properly since the method you're creating is called `of_age?` and not `is_of_age?`. Kindly alter your method name so this functionality can work properly
- [ ] It is a usually best practice to have private methods at the end of your class. Additionally, you don't have to imperatively specify `public` for your public methods since methods are public by default. Kindly remove this `public` command.